test/start.test.js

```diff
@@ -184,19 +196,19 @@ test('should start fastify with custom plugin options with a typescript compiled
   const fastify = await start.start(argv)
 
   const result = await fetch(`http://localhost:${fastify.server.address().port}`)
-  t.equal(result.status, 200)
+  t.assert.equal(result.status, 200)
 
   const body = await result.text()
-  t.equal(result.headers.get('content-length'), '' + body.length)
-  t.same(JSON.parse(body), {
+  t.assert.equal(result.headers.get('content-length'), '' + body.length)
+  t.assert.deepStrictEqual(JSON.parse(body), {
     a: true,
     b: true,
     c: true,
     hello: 'world'
   })
 
-  await fastify.close()
-  t.pass('server closed')
+  t.after(async () => await fastify.close())
```

Comment:
Here we need to wait for the server to close, otherwise the next `t.assert.ok` doesn't make any sense
```suggestion
  await fastify.close()
```